agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -1668,7 +1668,7 @@ bool backhaul_manager::handle_slave_backhaul_message(std::shared_ptr<SSlaveSocke
         LOG(DEBUG) << "ACTION_BACKHAUL_HOSTAP_VAPS_LIST_UPDATE_NOTIFICATION received from iface "
                    << soc->hostap_iface;
         if (m_agent_ucc_listener) {
-            auto msg = cmdu_rx.getClass<
+            auto msg = beerocks_header->addClass<
```

Comment:
Meh, if we ever do autoparsing of the Intel VS TLV, then we have to change this into getClass again...

Anyway, it clearly has to be on beerocks_header and not on cmdu_rx, so it's a step in the right direction (not to mention that it fixes the current issue :-)